stories/MonthPicker.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { storiesOf } from '@kadira/storybook';
+import moment from 'moment';
+import MonthPicker from '../src/components/MonthPicker';
+
+import { VERTICAL_ORIENTATION } from '../constants';
+
+const TestPrevIcon = () => (
+  <span
+    style={{
```

Comment:
Totally right, I have extracted the style into a variable to be able to reuse it.